agents/ppo/__init__.py

```diff
@@ -0,0 +1 @@
+from .ppo import PPOAgent, BaseActor, BaseCritic
```

Comment:
From my view `ppo/ppo.py` is a bad filename. It gives redundant or ubiquitous explanation. My idea: let's make either a single `ppo.py` file inside `agents` directory or create separated files for actor and critic respectively.

I suggest use the first option because for sure `types.py` is not a part of PPO agent. It should be put in the utils or types section.

Reply:
I agree that `types.py` should be on top level, I think that we may have several implementations of actor, critic and dto classes in one file) may be java code style talks in me and in our case this is ok)